file_system.go

```diff
@@ -112,6 +112,27 @@ func (c *Client) Getwd() (string, error) {
 	return dir, nil
 }
 
+// Setwd changes the current working directory.
+func (c *Client) Setwd(path string) (error) {
```

Comment:
I didn't include this because it isn't compatible with the connection pool (would end up with connections in different directories). I considered making it a client-wide setting that gets applied whenever you get a connection (i.e. switches to the desired directory), but that could get complicated, too.